src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Resources/Base/BuildReleaseArm64SimpleDotNet.apkdesc

```diff
@@ -5,79 +5,82 @@
       "Size": 3032
     },
     "assemblies/Java.Interop.dll": {
-      "Size": 58990
+      "Size": 58924
     },
     "assemblies/Mono.Android.dll": {
-      "Size": 87490
+      "Size": 87710
     },
     "assemblies/Mono.Android.Runtime.dll": {
-      "Size": 5860
+      "Size": 5924
     },
     "assemblies/rc.bin": {
       "Size": 1182
     },
     "assemblies/System.Console.dll": {
-      "Size": 6627
+      "Size": 6593
     },
     "assemblies/System.Linq.dll": {
-      "Size": 9252
+      "Size": 9253
     },
     "assemblies/System.Private.CoreLib.dll": {
-      "Size": 470830
+      "Size": 484001
     },
     "assemblies/System.Runtime.dll": {
-      "Size": 2626
+      "Size": 2629
     },
     "assemblies/System.Runtime.InteropServices.dll": {
-      "Size": 2267
+      "Size": 2269
     },
     "assemblies/UnnamedProject.dll": {
-      "Size": 3629
+      "Size": 3628
     },
     "classes.dex": {
       "Size": 18968
     },
+    "lib/arm64-v8a/libmono-component-marshal-ilgen.so": {
+      "Size": 93552
+    },
     "lib/arm64-v8a/libmonodroid.so": {
-      "Size": 434632
+      "Size": 434704
     },
     "lib/arm64-v8a/libmonosgen-2.0.so": {
-      "Size": 3076080
+      "Size": 3064488
     },
```

Comment:
@naricc does `libmono-component-marshal-ilgen.so` look like the expected file size? Does it seem like `libmonosgen-2.0.so` should get smaller? We lost 11kb, but gained 93kb?

Reply:
That looks like about the expected file size for `libmono-component-marshal-ilgen.so`. But I would expect the size of libmono-sgen-2.0.so to shrink more.

Reply:
@naricc should I file an issue in dotnet/runtime? It's not a large app size regression, but maybe someone should check we don't have duplicate methods in there or something.

Reply:
Yeah, I will look into it.